man/systemd.service.xml

```diff
@@ -1088,6 +1088,279 @@ ExecStart=/bin/echo $ONE $TWO $THREE</programlisting>
+    </para>
+    <table>
+      <title>systemd-specific exit codes</title>
+      <tgroup cols='3'>
+        <thead>
+          <row>
+            <entry>Exit Code</entry>
+            <entry>Symbolic Name</entry>
+            <entry>Description</entry>
+          </row>
+        </thead>
+        <tbody>
+          <row>
+            <entry>200</entry>
+            <entry>EXIT_CHDIR</entry>
+            <entry>The chdir(2) call failed.</entry>
+          </row>
+          <row>
+            <entry>201</entry>
+            <entry>EXIT_NICE</entry>
+            <entry>Failed to set the scheduling priority of a process.</entry>
+          </row>
+          <row>
+            <entry>202</entry>
+            <entry>EXIT_FDS</entry>
+            <entry>Closing up of file descriptors (TODO: Which ones??) failed.</entry>
+          </row>
+          <row>
+            <entry>203</entry>
+            <entry>EXIT_EXEC</entry>
+            <entry>The execve(2) call failed.</entry>
+          </row>
+          <row>
+            <entry>204</entry>
+            <entry>EXIT_MEMORY</entry>
+            <entry>Failed to perform an action due to memory shortage.</entry>
+          </row>
+          <row>
+            <entry>205</entry>
+            <entry>EXIT_LIMITS</entry>
+            <entry>Failed to set resoure limits.</entry>
+          </row>
+          <row>
+            <entry>206</entry>
+            <entry>EXIT_OOM_ADJUST</entry>
+            <entry>Failed to adjust the OOM score.</entry>
+          </row>
+          <row>
+            <entry>207</entry>
+            <entry>EXIT_SIGNAL_MASK</entry>
+            <entry>Failed to reset a program's signal mask.</entry>
+          </row>
+          <row>
+            <entry>208</entry>
+            <entry>EXIT_STDIN</entry>
+            <entry>Failed to set up a program's standard input.</entry>
+          </row>
+          <row>
+            <entry>209</entry>
+            <entry>EXIT_STDOUT</entry>
+            <entry>Failed to set up a program's standard output.</entry>
+          </row>
+          <row>
+            <entry>210</entry>
+            <entry>EXIT_CHROOT</entry>
+            <entry>The chroot(2) call failed.</entry>
+          </row>
+          <row>
+            <entry>211</entry>
+            <entry>EXIT_IOPRIO</entry>
+            <entry>Failed to set up a program's IO scheduler priority.</entry>
+          </row>
+          <row>
+            <entry>212</entry>
+            <entry>EXIT_TIMERSLACK</entry>
+            <entry>Failed to set up a program's timer slack.</entry>
+          </row>
+          <row>
+            <entry>213</entry>
+            <entry>EXIT_SECUREBITS</entry>
+            <entry>Failed to set up a program's secure bits.</entry>
+          </row>
+          <row>
+            <entry>214</entry>
+            <entry>EXIT_SETSCHEDULER</entry>
+            <entry>The sched_setscheduler(2) system call failed.</entry>
```

Comment:
and here (and for the rest too, i'll stop comment on this for the rest of the options). I think to be truly useful the table should mention for each exit code where it applies which config settings might be at fault